tests/connect/bug1045.phpt

```diff
@@ -12,7 +12,7 @@ PHPC-1045: Segfault if username is not provided for SCRAM-SHA-1 authMechanism
 require_once __DIR__ . "/../utils/basic.inc";
 
 // URI does not support auth, but that is not necessary for the test
-$m = new MongoDB\Driver\Manager(URI, ['authMechanism' => 'SCRAM-SHA-1', 'ssl' => false]);
+$m = new MongoDB\Driver\Manager(URI, ['authMechanism' => 'SCRAM-SHA-1']);
```

Comment:
Is the "URI does not support auth" comment above still relevant, given that we may be testing against a server using authentication?

Perhaps this should be changed to: "URI may or may not support auth, but that..."

Reply:
Updated